external-api/src/http/admin.rs

```diff
@@ -11,6 +11,12 @@ use serde::{Deserialize, Serialize};
 pub const IS_LEADER_ROUTE: &str = "/v0/admin/is-leader";
 /// Get the open orders managed by the node
 pub const ADMIN_OPEN_ORDERS_ROUTE: &str = "/v0/admin/open-orders";
+/// Route to create a matching pool
+pub const ADMIN_MATCHING_POOL_CREATE_ROUTE: &str = "/v0/admin/:matching_pool/create";
```

Comment:
i'd suggest we do `/v0/admin/matching-pools/:name/create`. Think about what this endpoint ends up being when we substitute in a concrete value, maybe we call a pool `order123`: `/v0/admin/order123/create`. It's unclear _what_ object I'm creating. & we shouldn't rely on pools being named with "pool" in them or something, that'd be an odd coupling.

Same for the other endpoints too to be clear.

Reply:
One nit here too: I'd remove `create` from the path. We typically think about "posting" to a path to be editing or creating the object at that path. This is in line with precedent elsewhere in the relayer's API (e.g. wallet creation) so it'd be good to be consistent. `destroy` is fine though, since it specifies a non-default post operation

Reply:
ack, i already make this change in https://github.com/renegade-fi/renegade/pull/700 but I'll remove the `/create` path segment and have it be a POST method